backend/app/analysis/sentiment_analysis.py

```diff
@@ -0,0 +1,35 @@
+import nltk
+from nltk.sentiment import SentimentIntensityAnalyzer
+import numpy as np
+
+
+def text_to_sound(user_text):
+    """
+    Takes text, runs NLTK's sentiment analyzer, and then returns a sound object
+    """
+
+    nltk_analyzer = SentimentIntensityAnalyzer()
+    all_sentences = nltk.sent_tokenize(user_text)
+
+    all_scores = []
+    for each_sentence in all_sentences:
+        all_scores.append(nltk_analyzer.polarity_scores(each_sentence)["compound"] * 15)
+
+    all_frequencies = []
+    for each_score in all_scores:
+        frequency = 440 * (2 ** (each_score / 12))
+        all_frequencies.append(frequency)
+
+    sample_rate = 44100
+    T = 1
+    t = np.linspace(0, T, T * sample_rate, False)
+
+    all_notes = []
+    for each_freq in all_frequencies:
+        all_notes.append(np.sin(each_freq * t * 2 * np.pi))
+
+    audio = np.hstack(all_notes)
+    audio *= 32767 / np.max(np.abs(audio))
+    audio = audio.astype(np.int16)
+
+    return audio, sample_rate
```

Comment:
This is a great start! I'd love to see a unit test around this function. I don't think we need (or meaningfully can?) test that the byte array that's returned contains the appropriate values (since there'll be...44100 values), but we can at least assert that the data that's returned is of the expected type, shape, and length. This'll help protect us in the future: if we make changes to the function, we can be sure that the function will still return expected types. A test we write now can therefore serve as a regression test for ourselves in the future.